source/renderer/app/stores/NetworkStatusStore.js

```diff
@@ -122,6 +124,8 @@ export default class NetworkStatusStore extends Store {
     // Passively receive state changes of the cardano-node
     cardanoStateChangeChannel.onReceive(this._handleCardanoNodeStateChange);
 
+    setLogStateSnapshotChannel.onReceive(this._handleLogStateSnapshot);
```

Comment:
@DeeJayElly please add a comment above this line explaining what it does

Reply:
Done